jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -191,6 +199,9 @@ def single_pipeline(job) {
         stage '020-install-pe'
         step020_install_pe(SKIP_PE_INSTALL, SCRIPT_DIR, server_era)
 
+        stage '025-collect-facter-data'
```

Comment:
That's a good idea - it hadn't occurred to me that the 'stage' part might be able to be inside of a conditional, but it's worth a try.